Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -90,9 +93,18 @@ internal final class SettingsNotificationsViewController: UIViewController {
 
   private func animatePickerView(isHidden: Bool) {
     UIView.animate(withDuration: 0.25) {
-      self.emailPickerViewTopConstraint.constant = isHidden
-        ? 0 : -SettingsNotificationsViewController.emailPickerViewHeight
 
+      if !isHidden {
+        // Tells Voice Over to focus on the picker.
+        self.emailFrequencyPickerView.accessibilityViewIsModal = true
```

Comment:
Can you check `MessageBannerViewController.swift` and add `isVoiceOverRunning()` check for consistency?

```
// Tells Voice Over to resign focus of the message banner.
          // This causes the reader to focus on the previously selected element.
          if AppEnvironment.current.isVoiceOverRunning() { ... }
```

Same for